test/gitStash.spec.js

```diff
@@ -0,0 +1,430 @@
+import path from 'path'
+import fsp from 'fs-promise'
+import tmp from 'tmp'
+import gitflow from '../src/gitWorkflow'
+
+jest.unmock('execa')
+tmp.setGracefulCleanup()
+
+let wcDir
+let wcDirPath
+let gitOpts = { cwd: 'test/__fixtures__' }
+const initialContent = `module.exports = {
+    test: 'test2'
+}
+`
+
+async function gitStatus(opts = gitOpts) {
+  return gitflow.execGit(['status', '--porcelain'], opts).then(res => res.stdout)
+}
+
+async function gitStashList(opts = gitOpts) {
+  return gitflow.execGit(['stash', 'list'], opts).then(res => res.stdout)
+}
+
+async function readFile(filepath, dir = wcDirPath) {
+  const content = await fsp.readFile(path.join(dir, filepath), { encoding: 'utf-8' })
+  return content
+}
+
+describe('git', () => {
+  beforeEach(async () => {
+    wcDir = tmp.dirSync({ unsafeCleanup: true })
+    wcDirPath = wcDir.name
+    gitOpts = {
+      cwd: wcDirPath,
+      gitDir: path.join(wcDirPath, '.git')
+    }
+
+    // Init repository
+    await gitflow.execGit('init', gitOpts)
+    // Create JS file
+    await fsp.writeFile(
+      path.join(wcDirPath, 'test.js'),
+      `module.exports = {
```

Comment:
Can we use `dedent` for these multi-line strings? Note that it would trim any leading or trailing white space including `\n`.